src/libraries/System.Private.CoreLib/src/System/Text/Unicode/Utf16Utility.Validation.cs

```diff
@@ -133,14 +137,20 @@ internal static unsafe partial class Utf16Utility
                         Vector128<ushort> charIsThreeByteUtf8Encoded;
                         uint mask;
 
+                        // Since 3-byte elements have a value >= 0x0800, we'll perform a saturating add of 0x7800 in order to
```

Comment:
It would be nice if there was a brief description of these next to the constants.